datakyt/db/test_project.csv

```diff
@@ -0,0 +1,11 @@
+id,name
```

Comment:
id is something that db should take care of. remove it from csv file

Reply:
Done, I checked we don't have any mistake if use csv file without id and name

Reply:
Do not see any changes